backend/danswer/chat/process_message.py

```diff
@@ -446,6 +486,13 @@ def stream_chat_message_objects(
                     tool_dict[db_tool_model.id] = [
                         ImageGenerationTool(api_key=dalle_key)
                     ]
+                elif tool_cls.__name__ == InternetSearchTool.__name__:
+                    bing_api_key = BING_API_KEY
+                    if not bing_api_key:
+                        raise ValueError("Internet search tool requires a Bing API key")
```

Comment:
Can maybe add "please contact your Danswer admin to get it added!" We always want users to have some actionable way to get unblocked.